src/lib/seam/connected-accounts/use-connected-account.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query'
+import type {
+  ConnectedAccount,
+  ConnectedAccountsGetRequest,
+  ConnectedAccountsGetResponse,
+  SeamError,
+} from 'seamapi'
+
+import { useSeamClient } from 'lib/seam/use-seam-client.js'
+import type { UseSeamQueryResult } from 'lib/seam/use-seam-query-result.js'
+
+export type UseConnectedAccountData = ConnectedAccount | null
+
+export function useConnectedAccount(
+  params: ConnectedAccountsGetRequest
+): UseSeamQueryResult<'connected_account', UseConnectedAccountData> {
```

Comment:
Ref https://github.com/seamapi/react/blob/main/src/lib/seam/access-codes/use-access-code.ts#L17C24-L17C34
```suggestion
): UseSeamQueryResult<'connectedAccount', UseConnectedAccountData> {
```